addons/l10n_in_pos/static/src/overrides/components/pos_receipt.xml

```diff
@@ -26,5 +26,29 @@
                 </div>
             </t>
         </xpath>
+        <xpath expr="//div[@class='before-footer']" position="after">
+            <br/>
+            <table t-if="props.data?.l10n_in_hsn_summary and props.data.headerData.company.country_id?.code === 'IN'" style="width:100%;">
+                <tr>
+                    <th class="text-center fw-bolder" colspan="6">--- HSN Summary ---</th>
+                </tr>
+                <tr>
+                    <th class="text-center">HSN Code</th>
+                    <th class="text-center">Rate%</th>
+                    <th class="text-center">CGST</th>
+                    <th class="text-center">SGST</th>
+                    <th class="text-center">IGST</th>
+                    <th class="text-center">CESS</th>
+                </tr>
+                <tr t-foreach="props.data.l10n_in_hsn_summary" t-as="line" t-key="line_index">
```

Comment:
```suggestion
                <tr t-foreach="props.data.l10n_in_hsn_summary" t-as="line_value" t-key="line_index">
```

Should it be line_value because in the <td> elements below, reference to `line_value` is needed? But I could be wrong.

Reply:
well it did concerned me earlier but, while using `line_value` in the below elements it displays as JSON values (it works correctly here)

Reply:
Oh shoot, I remember now. It's owl that makes these values available. With `t-as="line"`, the `line_index` and `line_value` references will be available.